crates/interpreter/src/instructions/data.rs

```diff
@@ -1,24 +1,24 @@
 use crate::{
     gas::{cost_per_word, BASE, DATA_LOAD_GAS, VERYLOW},
-    interpreter::Interpreter,
     interpreter_types::{
         EofData, Immediates, InterpreterTypes, Jumps, LoopControl, MemoryTr, RuntimeFlag, StackTr,
     },
     Host,
 };
 use primitives::{B256, U256};
 
+use super::context::InstructionContext;
+
 pub fn data_load<WIRE: InterpreterTypes, H: Host + ?Sized>(
-    interpreter: &mut Interpreter<WIRE>,
-    _host: &mut H,
+    context: &mut InstructionContext<'_, H, WIRE>,
```

Comment:
```suggestion
    ctx: &mut InstructionContext<'_, H, WIRE>,
```
Suggestion: Using `ctx` for the sake of naming consistency in rust.